style.css

```diff
@@ -1,136 +1,450 @@
-  background-repeat: no-repeat;
-  background-size: 40%;
-  background-position: 100% 120%;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 10rem;
+  padding-top: 2rem;
+  padding-bottom: 0.5rem;
+  padding-left: 0.5rem;
 }
 
-/*  headlines test  */
-.headline_texts {
-  height: 60%;
-  width: 85.5%;
-  margin: 7%;
-  position: relative;
-  top: 20%;
+#headline > ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 10rem;
+  padding-top: 2rem;
+  padding-bottom: 3.5rem;
+  padding-left: 0.5rem;
+}
+
+#headline > ul > li {
+  align-items: center;
 }
 
-/*  social media links  */
-.social_media {
+/* work selector */
+#work {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   justify-content: center;
-  align-items: center;
-  height: 11%;
-  width: 60%;
-  gap: 1ch;
-  list-style: none;
+  height: auto;
+  background: var(--txtcolor);
+  color: var(--txtcolor2);
+  padding: 0 2rem 3rem 2rem;
 }
 
-.social_media > img {
-  height: 50%;
-  width: 50%;
-  border-radius: 0;
+#work > h1 {
+  margin: 5rem 0;
+  font-size: 2.3rem;
+  font-family: 'Crete Round', serif;
+  font-weight: 500;
+  justify-items: center;
+  padding: 2rem 1rem 2rem 0;
+  border-bottom: 1.5px solid var(--line);
 }
 
-h1 {
-  position: static;
-  width: 327px;
-  height: 124px;
-  left: 24px;
-  top: 120px;
+#work > img {
+  padding-bottom: 2rem;
+}
+
+#work > h2 {
   font-family: 'Crete Round', serif;
-  font-style: normal;
+  font-weight: 300;
+  font-size: 2rem;
+}
+
+#work > p {
+  font-family: 'Inter', sans-serif;
+  padding-top: 0.5rem;
   font-weight: 400;
-  font-size: 56px;
-  line-height: 64px;
-  letter-spacing: 0.37px;
-  color: #ff6b00;
+  line-height: 1.5rem;
+  max-width: 20rem;
+  font-size: 1rem;
+  letter-spacing: 0;
+  text-align: left;
   flex: none;
-  order: 0;
+  font-style: normal;
   flex-grow: 0;
-  margin: 12px 0;
 }
 
-h3 {
-  position: static;
-  width: 327px;
-  height: 8%;
-  left: 24px;
-  top: 256px;
-  font-family: "Roboto", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 24px;
-  line-height: 32px;
+#work > ul {
+  list-style-type: none;
   display: flex;
+  flex-direction: row;
+  padding: 1rem 0;
+}
+
+#work > ul > li {
+  font-family: 'Inter', sans-serif;
+  border: 1px solid var(--line);
+  padding: 0.5rem;
+  margin-right: 0.5rem;
+}
+
+#work > button {
+  background: var(--maincolor);
+  border: none;
+  width: 7rem;
+  padding: 0.5rem;
+  font-family: 'Inter', sans-serif;
+  color: var(--txtcolor);
+  font-weight: 400;
+  font-size: 1rem;
+}
+
+/* work carts selector */
+#work > .cart {
+  display: grid;
+  grid-template-columns: 1;
+  grid-template-rows: 4;
+  margin-top: 2rem;
+  padding-top: 7rem;
+  height: 18.8rem;
+  width: 19.5rem;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  border: none;
+  background: url(Resources/Images/Img-Placeholder.png);
+  color: var(--txtcolor);
+}
```

Comment:
- [ ] - [ ] It is required that the content of the project is aligned with a flexbox. Please change it from grid to flexbox.